core/src/rpc.rs

```diff
@@ -382,6 +382,91 @@ impl JsonRpcRequestProcessor {
         Ok(result)
     }
 
+    pub fn get_inflation_reward(
+        &self,
+        addresses: Vec<Pubkey>,
+        config: Option<RpcEpochConfig>,
+    ) -> Result<Vec<Option<RpcInflationReward>>> {
+        let config = config.unwrap_or_default();
+        let commitment = Some(config.commitment.unwrap_or(CommitmentConfig::finalized()));
```

Comment:
I think we can delete this line now, and just pass `config.commitment` where `commitment` is used below

Reply:
This will 🔪 the clippy error too :)